### PR TITLE
Visitor can view all listed articles

### PR DESCRIPTION
**Pull request by johanbounce**
# [PT Story](https://www.pivotaltracker.com/story/show/171841161)
```
As a Visitor
In order to choose an article to read
I would like to be able to see the articles listed on a page
```
## Changes proposed in this pull request:
* Listing of all articles on componentDidMount
* Get request to API 

## What I have learned working on this feature: 
- Repitition is the mother of all learning
- Working with a new design package

## Packages/Gems added
- Axios
- Grommet

## Screenshots (Only if client side story)
Skärmavbild 2020-03-19 kl  18 04 25

### CODE REVIEW

File: cypress/fixtures/articles_index.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "articles": [
+    {
```
Comment (robin-lillqvist) [inline cypress/fixtures/articles_index.json]: include real formate of response json pls

File: src/components/ArticleList.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      });
+    });
+  }
+  render() {
+    let articleDisplay;
+    if (this.props.articleList !== []) {
+      articleDisplay = this.props.articleList.map(article => {
+        return (
+          <>
+            <Grommet full theme={grommet}>
```
Comment (robin-lillqvist) [inline src/components/ArticleList.jsx]: move this grommet tag down to return instead, now we're rendering one for each article. We don't need to do that.

**Review by robin-lillqvist — CHANGES_REQUESTED** [review]
Left two comments

File: cypress/fixtures/articles_index.json (hunk 1 above)
Comment (johanbounce) [reply cypress/fixtures/articles_index.json]: Done

File: src/components/ArticleList.jsx (hunk 2 above)
Comment (johanbounce) [reply src/components/ArticleList.jsx]: Done

**Review by robin-lillqvist — APPROVED** [review]

**Pull request merged**